daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/archive/DecodeV1Spec.scala

```diff
@@ -1050,6 +1050,7 @@ class DecodeV1Spec
                 Map(),
                 Map(),
                 EPrimCon(PCUnit),
+                Map(),
```

Comment:
Are you handling tests separately? If so, maybe worth tracking that somewhere? If not, I think you’re missing some tests for this.

Reply:
We are indeed missing tests ;)

Reply:
ah yes, it seems our only scala decoder test for interfaces just tests the version; I'll add a couple more tests for the components of an interface definition